database/kangu-product_2016-01-25.sql

```diff
@@ -0,0 +1,70 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4499
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.5.38)
+# Database: kangu-product
+# Generation Time: 2016-01-25 10:35:42 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table tbl_advert
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `tbl_advert`;
+
+CREATE TABLE `tbl_advert` (
+  `advert_id` int(11) NOT NULL AUTO_INCREMENT,
+  `advert_info` varchar(500) NOT NULL,
+  `advert_rating` int(2) NOT NULL,
+  `advert_price` int(10) NOT NULL,
+  `advert_spots` int(2) NOT NULL,
+  PRIMARY KEY (`advert_id`),
+  CONSTRAINT `fk_user_id` FOREIGN KEY (`advert_id`) REFERENCES `tbl_users` (`user_id`)
+) ENGINE=InnoDB DEFAULT CHARSET=latin1;
+
+
+
+# Dump of table tbl_users
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `tbl_users`;
```

Comment:
Change the name into 'tbl_user' to correspond to the singular naming convention used in the database structure.